Add documentation for R4 Condition diagnosis write

cmccormick42 opened the pull request:
![Screen Shot 2020-03-27 at 2 30 38 PM](https://user-images.githubusercontent.com/47217759/77793184-a3181c80-7037-11ea-84fc-8cb8a1fee02e.png)

![Screen Shot 2020-03-27 at 2 30 50 PM](https://user-images.githubusercontent.com/47217759/77793148-9693c400-7037-11ea-9ae9-93bb58b177dc.png)

nishanthnamana commented:
We should also update note for abatementDateTime, saying it is only supported when category is 'problem-list-item'.

And we need to update notes for fields like verification_status and clinical_status. As 'encounter-diagnosis' only supports certain values of verification_status and clinical_status. Basically, we need to specify 422 scenario's for these fields.

cmccormick42 commented on content/millennium/r4/general-clinical/condition.md:
```diff
@@ -121,6 +121,8 @@ _Implementation Notes_
 * Only the body fields mentioned below are supported. Unsupported fields will be ignored.
 * Modifier fields should not be provided, and will cause the transaction to fail.
+* `abatementDateTime` is not supported for Conditions with a category of `encounter-diagnosis`.
+* `encounter` is required for Conditions with a category of `encounter-diagnosis`.
```
Moved here https://github.com/cerner/fhir.cerner.com/pull/361/commits/99e0ffc31e3ee17a661a7fd7346c5b60c2a8385f

cmccormick42 commented:
> We should also update note for abatementDateTime, saying it is only supported when category is 'problem-list-item'.
> 
> And we need to update notes for fields like verification_status and clinical_status. As 'encounter-diagnosis' only supports certain values of verification_status and clinical_status. Basically, we need to specify 422 scenario's for these fields.

Moved the abatement note in previous commit/comment.

Added note for clinicalStatus https://github.com/cerner/fhir.cerner.com/pull/361/commits/d8930a04f46654fdd66854e2f39e2b2c3c635413#diff-abb99d2139d79f8a2a8f1d214f5f6882R51

Aren't we allowing the same verificationStatus codes for problems and diagnoses?

nishanthnamana commented:
> > We should also update note for abatementDateTime, saying it is only supported when category is 'problem-list-item'.
> > And we need to update notes for fields like verification_status and clinical_status. As 'encounter-diagnosis' only supports certain values of verification_status and clinical_status. Basically, we need to specify 422 scenario's for these fields.
> 
> Moved the abatement note in previous commit/comment.
> 
> Added note for clinicalStatus [d8930a0#diff-abb99d2139d79f8a2a8f1d214f5f6882R51](https://github.com/cerner/fhir.cerner.com/commit/d8930a04f46654fdd66854e2f39e2b2c3c635413#diff-abb99d2139d79f8a2a8f1d214f5f6882R51)
> 
> Aren't we allowing the same verificationStatus codes for problems and diagnoses?

+1, Yeah we are allowing the same verificationStatus. No need for any explicit verificationStatus notes.

nishanthnamana commented:
**Validated the creates section locally**
Clinical Status
<img width="1429" alt="diagnosis_write_clinicalstatus" src="https://user-images.githubusercontent.com/15035274/78705089-8f3fa680-78d2-11ea-89a6-7955ea8364f7.png">

Category
<img width="1390" alt="diagnosis_write_category" src="https://user-images.githubusercontent.com/15035274/78705110-98307800-78d2-11ea-8307-c96e1192f733.png">

Encounter
<img width="1371" alt="diagnosis_write_encounter" src="https://user-images.githubusercontent.com/15035274/78705121-9d8dc280-78d2-11ea-888e-ef328754442c.png">

AbatementDateTime
<img width="1416" alt="diagnosis_write_abatment" src="https://user-images.githubusercontent.com/15035274/78705151-a7afc100-78d2-11ea-9a5a-d5bfdd40c852.png">

sanschutz0808 commented:
Validated
<img width="1680" alt="Screen Shot 2020-04-07 at 9 29 22 AM" src="https://user-images.githubusercontent.com/27974425/78706552-e5ade480-78d4-11ea-9865-4173184911e1.png">
<img width="661" alt="Screen Shot 2020-04-07 at 1 36 51 PM" src="https://user-images.githubusercontent.com/27974425/78706583-f0687980-78d4-11ea-8ab6-d5a7b7ef1aa5.png">

ebaranowski commented:
Validated:

<img width="972" alt="Screen Shot 2020-04-07 at 5 24 59 PM" src="https://user-images.githubusercontent.com/42553712/78725273-c0c96980-78f4-11ea-94f8-4d31946ed8e9.png">
